### ISSUE TITLE
Working example?

### ISSUE THREAD

**Issue by spencerthayer** [carlweis/messenger#1]
Does this work? How secure is it?

**Comment by bearettoaks** [comment]
Yes it works. Download the code and try it out. It's as secure as your server where you run the chat server.

**Comment by spencerthayer** [comment]
Are the conversations encrypted? How are the keys stored?
